src/components/TextField/README.md

```diff
@@ -830,32 +830,30 @@ See Apple’s Human Interface Guidelines and API documentation about accessibili
 
 <!-- content-for: web -->
 
-### Structure
-
 Screen readers convey information about text fields automatically through native HTML.
 
 - Use the `disabled` prop to add the HTML `disabled` attribute to the text field.
 - Use the `readOnly` prop to add the HTML `readonly` attribute to the text field.
-- If you use the `type` prop, then some assistive technologies adapt the software keyboard to the current task. This helps merchants with mobility, vision, and cognitive issues to enter information more easily.
+- If you use the `type` prop, then some browsers and assistive technologies adapt the software keyboard to the current task. This helps users with mobility, vision, and cognitive issues to enter information more easily.
 
 Use the `id` prop to provide a unique `id` attribute value for the text field. If you don't provide an `id`, then the component generates one automatically. All text fields need to have unique `id` values.
 
 ### Labeling
 
-The `label` prop is required to convey the purpose of the checkbox to all merchants.
+The `label` prop is required to convey the purpose of the checkbox to all users.
 
-If there are separate visual cues that convey the purpose of the text field to sighted merchants, then the label can be visually hidden with the `labelHidden` prop.
+If there are separate visual cues that convey the purpose of the text field to sighted users, then the label can be visually hidden with the `labelHidden` prop.
 
 When you provide help text via the `helpText` prop or an inline error message via the `error` prop, the help or error content is conveyed to screen reader users with the `aria-describedby` attribute. This attribute causes the content to be read along with the label, either immediately or after a short delay.
 
-Use the `placeholder` prop to provide additional instructions. However, don’t rely on placeholders alone since the content isn’t always conveyed to all merchants.
+Use the `placeholder` prop only to provide formatting instructions that aren’t essential, as [users may find difficult or impossible to access placeholders](https://www.nngroup.com/articles/form-design-placeholders/).
```

Comment:
Related to this point, and the following DO examples - the content standards workgroup is still finalizing new content patterns for text fields. Currently the group is planning to recommend that no placeholder text is used unless the text field is related to searching or filtering (is this component ever used in that context?), and that formatting instructions are included as help text instead. Here are the WIP guidelines if you're curious https://docs.google.com/document/d/1eNHv13nrJ562TN-f2UpvZmhstFRh-x_6wzPbgEzQz3s/edit# 

Can we align with that approach here as well?
PS I like having the NN group link there for additional context.

Reply:
@sadiesaurus  Text fields could totally be used for a search or filter interface. What type of text would the search or filter examples use, just "Search" or "Filter" as a visual prompt to duplicate a visually hidden label?

I also really like the phrase "Don’t use placeholder text for comments and notes; use help text instead" in the doc you shared.

Reply:
Yes, the pattern that we're proposing is going to be "Search {x}" or "Filter {x}" (for example "Search themes" or "Filter orders"). I think the plan is to recommend that these placeholders are floating when you type, like here in the Help Center search https://help.shopify.com/